evm/src/main/java/org/hyperledger/besu/evm/processor/ContractCreationProcessor.java

```diff
@@ -175,6 +175,9 @@ public void codeSuccess(final MessageFrame frame, final OperationTracer operatio
             contractCode.size(),
             frame.getRemainingGas());
         frame.setState(MessageFrame.State.COMPLETED_SUCCESS);
+        if (operationTracer.isExtendedTracing()) {
```

Comment:
Why only in extended?  Why not log this account creation all the time?

Reply:
It was not logged before, only in the failure cases. So it would change the existing behavior for anybody using the current tracer implementation. I think it makes sense to always log it though. So we might just change it to always log it.

Reply:
All the current tests are passing, so there is no problem in general. Do you think it makes sense to log it all the time? I am not sure why we did not do in the first place. But that might just have been an oversight.